src/rdkafka_mock_handlers.c

```diff
@@ -55,6 +55,7 @@ static int rd_kafka_mock_handle_Produce(rd_kafka_mock_connection_t *mconn,
         int16_t Acks;
         int32_t TimeoutMs;
         rd_kafka_resp_err_t all_err;
+        rd_kafka_mock_broker_t *leader = NULL;
```

Comment:
Upgrade max Produce and Fetch versions in INTRODUCTION.md
and add KIP-951 to the list of supported KIPs